src/config.ts

```diff
@@ -39,14 +49,31 @@ const defaultConfig: Config = {
   setupProvider: async baseProvider => baseProvider,
 
   coverage: false,
+
+  node: {
+    allowUnlimitedContractSize: false,
+    gasLimit: DEFAULT_BLOCK_GAS_LIMIT,
+    gasPrice: 20e9, // 20 gigawei
```

Comment:
What does this option do? Is it a minimum gas price for the node to accept the transaction?

Reply:
No, it is:
> Sets the default gas price for transactions if not otherwise specified.
We are consolidating all ganache settings under `node` option.

Reply:
Ah I see, thanks! I got this mixed up with `defaultGas`, which works a bit differently (you _need_ to specify it for contracts).